src/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Contenta Angular",
-  "short_name": "Contenta",
+  "name": "Umami Magazine",
```

Comment:
Yes. They plan to have a "Powered by Contenta" footer item. They mentioned in Slack yesterday.